src/OrchardCore.Modules/OrchardCore.AutoSetup/AutoSetupMiddleware.cs

```diff
@@ -10,6 +10,7 @@
 using OrchardCore.Abstractions.Setup;
 using OrchardCore.AutoSetup.Extensions;
 using OrchardCore.AutoSetup.Options;
+using OrchardCore.Data;
```

Comment:
Why this is added?

Reply:
We needed it at some point, I think we can remove it, also an additional project ref in the csproj file.

I will check it